torch/ao/quantization/fx/_decomposed.py

```diff
@@ -965,6 +869,8 @@ def dequantize_per_channel_group(
     return w_dq
 
 
+quantized_decomposed_lib = Library(ns, "DEF")
+
 quantized_decomposed_lib.define(
     "fake_quant_per_channel(Tensor input, Tensor scales, Tensor zero_points, int axis, "
     "int quant_min, int quant_max) -> Tensor")
```

Comment:
Should we add a TODO for this one to migrate it too? Maybe explain in the comment why we can't do this now?

Reply:
Yes, I'll add a TODO